airbyte-api/build.gradle

```diff
@@ -98,6 +98,18 @@ task generateApiDocs(type: GenerateTask) {
             generatePom  : "false",
             interfaceOnly: "true"
     ]
+
+    doLast {
```

Comment:
would it make sense to output the docs there by default instead of doing a copy?

Reply:
mostly trying to keep the pattern the same where we generate to the build dir, and then separately copy it where it needs to go. i think building to something outside of module is not great. i'm also not sure the gen allows it, but i can double check.